cli/tests/pcluster/api/controllers/test_cluster_operations_controller.py

```diff
@@ -753,7 +771,22 @@ def _send_test_request(self, client, cluster_name="clustername", region="us-east
                     ],
                     "version": get_installed_version(),
                     "scheduler": {"type": "slurm"},
-                    "details": [{"alarmType": "test_ProtectedModeAlarm_HeadNode", "alarmState": "ALARM"}],
+                    "details": [
```

Comment:
The details here should not be a list anymore

Reply:
Change it to a object. Thanks!